packages/orbit-components/src/Select/index.tsx

```diff
@@ -64,146 +64,153 @@ const Select = React.forwardRef<HTMLSelectElement, Props>((props, ref) => {
   const shown = tooltipShown || tooltipShownHover;
 
   return (
-    <label
-      className={cx("relative block", spaceAfter && spaceAfterClasses[spaceAfter])}
-      ref={inputRef}
-      style={{ width }}
-    >
-      {label && !inlineLabel && (
-        <FormLabel
-          error={!!error}
-          help={!!help}
-          labelRef={labelRef}
-          iconRef={iconRef}
-          onMouseEnter={() => setTooltipShownHover(true)}
-          onMouseLeave={() => setTooltipShownHover(false)}
-          required={required}
-        >
-          {label}
-        </FormLabel>
-      )}
-      <div
-        ref={label ? null : labelRef}
-        className={cx(
-          "orbit-select-container",
-          "relative flex flex-row items-center justify-between",
-          "h-form-box-normal box-border w-full",
-          disabled
-            ? "text-form-element-disabled-foreground cursor-not-allowed"
-            : "text-form-element-filled-foreground cursor-pointer",
-          !disabled &&
-            (error
-              ? "[&_.orbit-input-field-fake-input]:hover:shadow-form-element-error-hover"
-              : "[&_.orbit-input-field-fake-input]:hover:shadow-form-element-hover"),
-          "focus-within:outline-none",
-          "[&_.orbit-input-field-fake-input]:focus-within:outline-blue-normal [&_.orbit-input-field-fake-input]:focus-within:outline [&_.orbit-input-field-fake-input]:focus-within:outline-2",
-        )}
+    <>
```

Comment:
This seemed like the best solution to solve this issue without refactoring the component - the ideal solution would probably be to use explicit label and label would only contain the label text. Currently the `aria-label` for the help/error icon is still present in the label but I think it's not such a problem as having tooltip content be announced twice. These `aria-labels` for icons should be also translated, however.